Homework lesson9 added.

iBubbleGun opened the pull request:
Added homework lesson9.
Please, check it.

iBubbleGun committed:
Start homework lesson 9.

iBubbleGun committed:
Mostly finished homework lesson 9.

iBubbleGun committed:
Added homework lesson 9.

iBubbleGun committed:
Added homework lesson 9.

kartius commented on src/main/java/edu/hillel/homework/lesson9/warehouse/fruit_storage_area/Lemon.java:
```diff
@@ -0,0 +1,11 @@
+public class Lemon extends Fruit {
+
+    public Lemon() {
+        final float weight = 0.8f;
```
One small detail - better to init weight as a static global parameter in the class. It's a constant

iBubbleGun replied:
Yes, I originally thought the same, but at the last moment I decided to insert it into the constructor.
And my decision was influenced first of all by the fact that for some reason the IDE often offers me to initialize such constants in the constructor.
Although I absolutely agree logically, the constant looks prettier when it is displayed at the beginning of the class as a global static field.

iBubbleGun committed:
Update homework lesson 9.
Move each fruits weight fields from constructors in to static constant field.

iBubbleGun committed:
Update homework lesson 9.
Move each fruits weight fields from constructors in to constant public static field.
